src/core/server/http/http_config.ts

```diff
@@ -73,6 +73,11 @@ export const config = {
       host: schema.string({
         defaultValue: 'localhost',
         hostname: true,
+        validate(value) {
+          if (value === '0') {
+            return 'value 0 is not a valid hostname';
```

Comment:
Knowing there was some previous usage of "0". Should we add additional help to the error text? `value 0 is not a valid hostname. Use "0.0.0.0" to bind to all interfaces.`

Reply:
Fixed 👍